rust/Makefile.am

```diff
@@ -15,7 +15,7 @@ if HAVE_RUST_VENDOR
 FROZEN = --frozen
 endif
 
-if !DEBUG
+if !RUST_DEBUG
```

Comment:
Would it be useful to enable RUST_DEBUG even if DEBUG? Or would you prefer them kept separate?

Reply:
Yeah I fixed this in the final merged version